usl_pipeline/tests/readers/test_elevation.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from rasterio import CRS
+
+from usl_pipeline.readers.elevation import read_from_geotiff
+from usl_pipeline.shared.entities import ElevationHeader
+
+
+def test_load_elevation_from_geotiff_default_no_data():
+    test_file_path = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "test_data.tif"
+    )
+    with open(test_file_path, "rb") as fd:
+        elevation = read_from_geotiff(fd)
+        assert elevation.header == ElevationHeader(
+            col_count=2,
+            row_count=2,
+            x_ll_corner=587069.5669010617,
+            y_ll_corner=4505948.133973204,
+            cell_size=2.0,
+            nodata_value=0.0,
+            crs=CRS({"init": "EPSG:32618"}),
+        )
+        assert elevation.data == [
```

Comment:
Can use
https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html
for numpy arrays

Reply:
Done